### PR TITLE
add useful tools list and add values page

### PR DESCRIPTION
**Pull request by galexrt**
Signed-off-by: Alexander Trost <galexrt@googlemail.com>

### CODE REVIEW

**Review by gauravsitlani — DISMISSED** [review]

File: docs/values.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+Why should you be authentic? Every one has their own skills and ways of doing things.
+
+We can support our customers and ourselves best, through our individual experiences and skills.
+Being honest with ourselves and our customers to build a relationship that can accept
+Trust. Being clear on what we can do for each other.
+
+## Empowerment
+
+Empower yourself, your colleagues and our customers by helping with your experience and knowledge.
```
Comment (ideepika) [inline docs/values.md:19]:
```suggestion
* Thinking in Open
  Empower yourself, colleagues and customers by thinking in the open.
```

File: docs/values.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+title: "Values"
+---
+
+This page describes our 3 core values including a "why" each value is important to us.
+
+## Authenticity
+
+Be authentic in what you are doing.
+
+Why should you be authentic? Every one has their own skills and ways of doing things.
```
Comment (ideepika) [inline docs/values.md:11]:
```suggestion
Why should you be authentic?
Everyone has their own skills and ways of doing things.
```

File: docs/values.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
```
Comment (ideepika) [inline docs/values.md:12]:
```suggestion
```

File: docs/values.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+We can support our customers and ourselves best, through our individual experiences and skills.
+Being honest with ourselves and our customers to build a relationship that can accept
+Trust. Being clear on what we can do for each other.
+
+## Empowerment
+
+Empower yourself, your colleagues and our customers by helping with your experience and knowledge.
+
+We need to share our experience and knowledge with each other to benefit our goal of helping our customers and the communities.
```
Comment (ideepika) [inline docs/values.md:21]:
```suggestion
* We need to share our experiences and knowledge openly with each other to empower ourselves in our goal of helping our customers and the communities.
```

File: docs/values.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+* Ownership of problems and situations -> Empower yourself by owning up to failure but also success, transparent to the customer.
```
Comment (ideepika) [inline docs/values.md:23]:
```suggestion
* Taking Ownership Of Problems And Situations
  Empower yourself by being decisive and owning up to failures and success.
```

**Review by ideepika — APPROVED** [review]
Values looks good! some optional suggestions I have added, thanks!

**Review by gauravsitlani — APPROVED** [review]

**Pull request merged**